src/System.Net.Ping/src/System.Net.Ping.csproj

```diff
@@ -12,6 +12,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uap'">
+    <NoWarn>$(NoWarn);414</NoWarn>
```

Comment:
This is needed because the new UWP partial _Ping_ class implementation doesn't use any of the _Ping_ class' member variables, leading to "not used" warnings.